src/utils/global_vars.py

```diff
@@ -87,21 +87,6 @@
 OUTSIDE_VALUE: int = 1
 """Threshold option for outside color"""
 
-OTSU_THRESHOLD_FILTER: sitk.OtsuThresholdImageFilter = sitk.OtsuThresholdImageFilter()
-"""Global OTSU filter"""
-BINARY_THRESHOLD_FILTER: sitk.BinaryThresholdImageFilter = (
-    sitk.BinaryThresholdImageFilter()
-)
-"""Global Binary filter"""
-LOWER_THRESHOLD: int = 100
-"""Threshold option for binary threshold"""
-UPPER_THRESHOLD: int = 200
-"""Threshold option for binary threshold"""
-INSIDE_VALUE: int = 0
-"""Threshold option for inside color"""
-OUTSIDE_VALUE: int = 1
-"""Threshold option for outside color"""
-
```

Comment:
For some reason, these lines were duplicated. No idea how this got in there, maybe came from when I was resolving merge conflicts